app/scripts/controllers/permissions/specifications.js

```diff
@@ -162,6 +162,21 @@ export const getPermissionSpecifications = ({
         }
       },
     },
+    [PermissionKeys['wallet_getLocale']]: {
+      permissionType: PermissionType.RestrictedMethod,
```

Comment:
If `permissionType` is `restrictedMethod` why do we need `if(hasPermission...` in the rpc method implementation (i.e. handlers/get-locale.js below) @rekmarks